docs/site/todo-tutorial-juggler.md

```diff
@@ -1,15 +1,15 @@
 ---
 lang: en
-title: 'Adding legacy juggler'
+title: 'Adding juggler bridge'
 keywords: LoopBack 4.0, LoopBack 4
 sidebar: lb4_sidebar
 permalink: /doc/en/lb4/todo-tutorial-juggler.html
-summary: LoopBack 4 Todo Application Tutorial - Adding legacy juggler
+summary: LoopBack 4 Todo Application Tutorial - Adding juggler bridge
 ---
 
-### Adding Legacy Juggler
+### Adding Juggler Bridge
 
-The Legacy Juggler is a "bridge" between the existing
+The Juggler Bridge is a "bridge" between the existing
 [loopback-datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler)
 and the new LoopBack 4 architecture. It provides the capabilities required to
```

Comment:
Weren't you also planning on removing the word `new` here?

Reply:
yes will do that